physionet-django/user/test_views.py

```diff
@@ -244,7 +244,7 @@ def test_edit_emails(self):
 
         # Test 5: Verify the newly added email
         # Get the activation info from the sent email
-        uidb64, token = re.findall('http://testserver/verify/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/',
+        uidb64, token = re.findall('http://localhost:8000/verify/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/',
```

Comment:
Please could you briefly explain this change Felipe?

Reply:
It simply sets the proper URL for the tests as added in the Sites fixtures.